File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k = 1;
        int count = 1;

        for (int i = 1; i < nums.length; i++){
            if (nums[i] == nums[i-1]){
                count++;
            } // same number
            else{
                count = 1;
            }// new number

            if (count <= 2){
                nums[k] = nums[i];
                k++;
            }

        }
        return k;
    }
}

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        ptr = 1
        counter = 1

        for i in range(1,len(nums)):
            if nums[i] == nums[i-1]:
                counter += 1
            else:
                counter = 1

            if counter <= 2:
                nums[ptr] = nums[i]
                ptr += 1

        return ptr
